content/03.prior-knowledge.cascadelearning.md

```diff
@@ -0,0 +1,30 @@
+
+Sometimes, due to paucity of data and its unstable predictive capabilities, one may feel the need to consult various machine learning methods and the predictions there-of to make an educated decision about the identified predictive patterns, collectively called _ensemble learning_. 
+In an ensemble learning workflow, one may use various parameters to maximize the learned information. 
+A consensus approach is often ideal to select predictive features from a group of independently running models, other times one may coordinate various learning methods in such a way that one model uses information from another model in augmenting its predictive capability. 
+Random forest algorithms are a great example of ensemble learning where a consensus of the features selected through many different and independent decision trees are considered and ranked to make a final ranked list of important features that are predictive of a certain outcome (e.g. disease) [@doi:10.1186/1472-6947-13-134, @doi:10.1023/A:1010933404324].
+For the ensemble of decision trees, e.g. random forest, bootstrap aggregation (or _bagging_) helps in forming the consensus[@doi:10.1214/aos/1031689014, @doi:10.1177/2045894019890549, @doi:10.1016/S0031-3203(02)00169-3].
+However, successful application of consensus based ensemble learning algorithms requires what is known as "gold standard" data. 
+These generally refer to well labeled cases where the diagnosis or label of a data point in the training dataset has very little uncertainty (or "label-noise") associated with it [@doi:10.1093/jamia/ocw028]. 
+In most cases of rare disease, due to the inherent nature of being less defined, the symptoms as well as any underlying biology comes with a reasonable amount of uncertainty (or "label-noise") leading to a _silver standard_ dataset[@doi:10.1109/TNNLS.2013.2292894, @doi:10.1093/jamia/ocw028].
+This kind of data begets modifications in ensemble learning methods where one method leverages findings from another method to capture stable patterns existing in the _silver standard_ data and reduce uncertainty.
+Such methods fall into the category of cascade learning.
+Among classifiers, cascade learning has been widely used in image recognition where initially a small subset of image features are used to classify images (e.g. features identifying a face like eyes, nose, mouth). 
+The initial classification is then augmented by more complex features and algorithms like AdaBoost ( _boosting_ ) that weight the various features are implemented to detect the content of the image (e.g. features identifying a human face like relative distance between eyes etc.)[@doi:10.1109/CVPR.2001.990537, @doi:10.1007/978-3-540-75175-5_16, @doi:10.1109/ICPR.2004.1334680].
+
+In rare diseases, a variant of cascade learning that showed robustness in view of uncertainty in the data was implemented to identify rare disease patients from electronic health records from the general population [@pmid:30815073].
```

Comment:
What do you mean by `in view of uncertainty` here?

Reply:
This was to refer to the data labels in rare disease that had uncertainty associated with them (label-noise). However, I added in the previous lines later on and now this sentence looks out of context... need to smoothen it out a bit :)